### ISSUE TITLE
Best way to display status information

### ISSUE THREAD

**Issue by plbowers** [lloeki/ex-mode#168]
I'm implementing a `:regs` (`:registers`) and need to be able to display many lines. In vim this occurs within the special ex-command section at the bottom and it expands as needed. But I've looked through the code a couple times and I don't see how I would go about doing this...

Sorry for a potentially newb question - I'm just getting into atom/vim-mode/ex-mode development...

**Comment by jazzpi** [comment]
You're right, the section at the bottom doesn't exist. I think the best way to display this sort of information in Atom is using the built-in [Notifications](https://atom.io/docs/api/v1.10.2/Notification), so you could for example call `atom.notifications.addInfo(registersText)`. Also see the [NotificationManager](https://atom.io/docs/api/v1.10.2/NotificationManager) docs.
